index.html

```diff
@@ -0,0 +1,130 @@
+        <h1 class="title">PYLADIES NAMIBIA TECH WORKSHOP 2023</h1>
+        <p class="info-desc">  ​​​​​​The workshop for Namibian women in technology to acquire and practice new tech skills
+            and soft skills and get a chance to apply them to real life projects is taking place in June, in Windhoek.
+             
+        </p>
+        <p class="date">2023.06.16(FRI) ~ 18(MON)</p>
+        <p class="location">@ Safari Court Hotel and Conference Centre, Windhoek, Namibia</p>
+    </Section>
+
+    <!-- Program Section -->
+    <section class="program">
+        <h3 class="program-title">Main Program</h3>
+        <hr>
+        <div class="program-cards">
+            <div class="card lecture">
+                <img src="images/program_icon_01.png" class="icon lecture-icon" alt="lecture" />
+                <h4 class="card-title title-lecture">Lecture</h4>
+                <p class="program-desc">Listen to speakers from various institutions and companies about the messages of sharing and opening.</p>
+            </div>
+            <div class="card web-design">
+                <img src="images/program_icon_02.png" class="icon web-design-icon" alt="Web Design" />
+                <h4 class="card-title title-web-design">Web Design</h4>
+                <p class="program-desc">Focus on website building, Design, Publication and everything else  you need to launch your own website.</p>
+            </div>
+            <div class="card mobile-app">
+                <img src="images/program_icon_03.png" class="icon mobile-icon" alt="Mobile App" />
+                <h4 class="card-title title-mobile app">Mobile App</h4>
+                <p class="program-desc">Ladies will design, build and program their own mobile app using the knowledge they got from the discussions.</p>
+            </div>
+            <div class="card discussion">
+                <img src="images/program_icon_04.png" class="icon discussion-icon" alt="Discussion" />
+                <h4 class="card-title title-discussion">Discussion</h4>
+                <p class="program-desc">Discussions with feature leading technology experts and professionals from all over the country.</p>
+            </div>
+            <div class="card community">
+                <img src="images/program_icon_05.png" class="icon community-icon" alt="Community" />
+                <h4 class="card-title title-community">Community</h4>
+                <p class="program-desc">Introduction and Registration to join the communities and technology groups from across the country by experts.</p>
+            </div>
+            <button type="submit" class="button-1">Join PyLadies Namibia 2023</button>
+        </div>
+        <a class="link-1" href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <!-- Featured Speakers Section-->
+    <section class="speakers">
+        <h2 class="speaker-title">Featured Speakers</h2>
+        <hr>
+        <div class="speakers-container"></div>
+
+    </section>
```

Comment:
Kindly introduce some margin around this section to match the given design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190583334-a7c61bae-2d13-4d6c-adb3-23e1e6500565.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190583468-f2fb9429-2966-4ee7-b5ca-366e72a7a9c1.png)